app/views/root/index.html.erb

```diff
@@ -48,9 +48,6 @@
 
       <section class="services-block" aria-labelledby="services-and-information-label" id="services-and-information">
         <div class="inner-block floated-children">
-          <div class="two-column-heading">
-            <h2 id="services-and-information-label">Services and information</h2>
```

Comment:
There is a precedent for 2), in the section directly below there is a hidden "Deparments and policy" heading:
`<h2 id="departments-and-policy-label" class="visuallyhidden">Departments and&nbsp;policy</h2>`